terraform/modules/emrfs-lambda/policy-munge-lambda-files/lambda_handler.py

```diff
@@ -246,12 +246,24 @@ def remove_existing_user_policies(role_name, all_policy_list):
 def create_policies_from_dict_and_return_list_of_policy_arns(dict_of_policy_name_to_munged_policy_objects):
     list_of_policy_arns = []
     for policy in dict_of_policy_name_to_munged_policy_objects:
+        prevent_matching_sids(policy.get('Statement'))
```

Comment:
This code is really confusing. So statement is actually an array of statements?

Reply:
This is how IAM policies are structured - `{"Version": "2012-10-17", "Statement": [<statement1>, <statement2> ...]}`

Reply:
No, you decided to name the key 'statement' instead of 'statements' in the dict as described in the comment header :-)